### PR TITLE
chore : update packages to latest in base image

### PR DESCRIPTION
**Pull request by Rajpratik71**
Update packages to the latest in the base image to get the latest fixes and reduce Vulnerability

Signed-off-by: Pratik Raj <rajpratik71@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -230,7 +230,9 @@ COPY --from=build-full /out /
 
 FROM ubuntu:${UBUNTU_VERSION} AS base
 # fuse3 is required by stargz snapshotter
+ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
+  apt-get dist-upgrade -y && \
```
Comment (AkihiroSuda) [inline Dockerfile:235]: This is unusual in Dockerfiles.
Better to just wait for `ubuntu:22.10` on Docker Hub to be updated

**Comment by fahedouch** [comment]
By automtic upgrade we lost control on image deps. As @AkihiroSuda said, it is better to follow Ubuntu release process

**Comment by Rajpratik71** [comment]
ok @AkihiroSuda  @fahedouch 

But this will help reduce the vulnerabilities of the image by updating the packages in every new release which got affected.

Updating the base image approach adds a lot of wait time for fixes.

**Comment by AkihiroSuda** [comment]
I'm rather planning to pin all the deb packages with https://github.com/reproducible-containers/repro-get for reproducibility, so let me close this.

Users may run `apt-get dist-upgrade` locally.

**Pull request closed without merging**